invoicepilot_app/serializers.py

```diff
@@ -23,10 +23,19 @@ class Meta:
         model = PaymentTransaction
         fields = '__all__'
 
-class InvoiceItemSerializer(serializers.ModelSerializer):
```

Comment:
It looks like there's an issue here, You've defined InvoiceItemSerializer twice. Kindly explain your reasoning behind this, and does this logic work when you test it on your end?